### ISSUE TITLE
Rewards : UX - A11Y : Support for keyboard interactions instead of mouse, especially for ads

### ISSUE THREAD

**Issue by alexwykoff** [brave/brave-browser#4261]
## Description 
Users from the field have reported issues with interactions requiring mouse clicks. They have asked for keyboard interactions to support their ability.

## Steps to Reproduce
   1.  Turn on Ads
   2. Try to interact with an ad without using a mouse

## Actual result:
Users cannot interact with the notification

## Expected result:
Users should be able to interact with the notification

## Reproduces how often: 
100%

## Brave version (brave://version info)
All

## Version/Channel Information:
All

- Can you reproduce this issue with the current release? yes
- Can you reproduce this issue with the beta channel? yes
- Can you reproduce this issue with the dev channel? yes
- Can you reproduce this issue with the nightly channel? yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields?
- Does the issue resolve itself when disabling Brave Rewards?
- Is the issue reproducible on the latest version of Chrome?

**Comment by NejcZdovc** [comment]
@alexwykoff we are using native notifications, so afaik there is nothing we can do about this. It's how OS support for A11Y is for notifications. Closing for now and we can reopen if we find a way or when we implement new notifications, which ever is faster

cc @tmancey

**Comment by tmancey** [comment]
@alexwykoff This would be controlled via the OS, some users on macOS are using workarounds such as https://akrabat.com/keyboard-control-of-macos-notification-alerts/
